recipes-bsp/barebox/barebox.inc

```diff
@@ -73,6 +73,13 @@ do_compile () {
 
 	fi
 
+	# If there is a '${B}/firmware' directory, use that instead of
+	# '${S}/firmware' in CONFIG_EXTRA_FIRMWARE_DIR
+	if [ -d ${B}/firmware ]; then
+		sed -i -e "s,^\(CONFIG_EXTRA_FIRMWARE_DIR=\"\)firmware\"$,\1${B}/firmware\"," \
```

Comment:
The default is `CONFIG_EXTRA_FIRMWARE_DIR=firmware`, so this breaks every user that copies stuff into the source directory. I don't see how we can avoid that, so I'd prefer this be opt-in. What about a `PACKAGECONFIG`?

Reply:
Note that the path is not changed if `${B}/firmware/ ` does not exist. So yes, people that copy to `${S}/firmware` as well as `${B}/firmware` are out of luck.